src/Microsoft.Azure.WebJobs.Host/Bindings/BindingTemplateExtensions.cs

```diff
@@ -46,23 +46,15 @@ public static void ValidateContractCompatibility(this BindingTemplate bindingTem
         /// <param name="bindingTemplate">The binding template to validate.</param>
         /// <param name="bindingData">The binding data to apply to the template.</param>
         /// <returns>The bound template string.</returns>
+        [Obsolete("Call instance method directly")]
         public static string Bind(this BindingTemplate bindingTemplate, IReadOnlyDictionary<string, object> bindingData)
```

Comment:
Good catch. The extensions compiled because they just bound to this obsolete version (no warnings). I'll make the new instance method public. 

---
In reply to: [114632812](https://github.com/Azure/azure-webjobs-sdk/pull/1117#discussion_r114632812) [](ancestors = 114632812)